### PR TITLE
test: Modify process.go and config.go to handle Firecracker

### LINKED ISSUE #1096: test: Modify process.go and config.go to handle Firecracker
We need to add Firecracker as a hypervisor in order that the docker functional and integration tests work.

### PR DESCRIPTION
**Pull request by GabyCT**
We need to add Firecracker as a hypervisor in order that the
docker functional and integration tests work.

Fixes #1096

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: process.go [hunk 1]
```diff
@@ -53,17 +53,24 @@ func processRunning(regexps []string) bool {
...
+	FirecrackerHypervisor: string(".*--api-sock.*" + containerID + ".*firecracker.sock.*"),
 	}
-	hypervisorRegexps := []string{hypervisorPath + ".*-name.*" + containerID + ".*-qmp.*unix:.*/" + containerID + "/.*"}
-	return processRunning(hypervisorRegexps)
+	for h, r := range typeHypervisor {
+		config, ok := KataConfig.Hypervisor[h]
+		if ok {
+			return processRunning([]string{config.Path + r})
+		}
+	log.Fatal("Could not determine if hypervisor is running")
```
Comment (devimc) [inline process.go]: missing `}`

File: process.go [hunk 2]
```diff
@@ -53,17 +53,24 @@ func processRunning(regexps []string) bool {
 
 // HypervisorRunning returns true if the hypervisor is still running, otherwise false
 func HypervisorRunning(containerID string) bool {
-	hypervisorPath := KataConfig.Hypervisor[DefaultHypervisor].Path
-	if hypervisorPath == "" {
-		log.Fatal("Could not determine if hypervisor is running: hypervisor path is empty")
-		return false
+	var typeHypervisor = map[string]string{
+	DefaultHypervisor: string(".*-name.*" + containerID + ".*-qmp.*unix:.*/" + containerID + "/.*"),
```
Comment (devimc) [inline process.go]: please run `gofmt`

**Comment by GabyCT** [comment]
@devimc changes applied

File: process.go [hunk 3]
```diff
@@ -53,17 +53,25 @@ func processRunning(regexps []string) bool {
 
 // HypervisorRunning returns true if the hypervisor is still running, otherwise false
 func HypervisorRunning(containerID string) bool {
-	hypervisorPath := KataConfig.Hypervisor[DefaultHypervisor].Path
-	if hypervisorPath == "" {
-		log.Fatal("Could not determine if hypervisor is running: hypervisor path is empty")
-		return false
+	var typeHypervisor = map[string]string{
+		DefaultHypervisor:     (".*-name.*" + containerID + ".*-qmp.*unix:.*/" + containerID + "/.*"),
```
Comment (jodh-intel) [inline process.go]: Nit: those brackets aren't required on this line and the line below.

**Comment by jodh-intel** [comment]
lgtm

**Pull request merged**